Formula/r/robots.rb

```diff
@@ -0,0 +1,39 @@
+class Robots < Formula
+  desc "Google's robots.txt parser and matcher as a C++ library (compliant to C++11)"
+  homepage "https://github.com/google/robotstxt"
+  url "https://github.com/google/robotstxt/archive/455b1583103d13ad88fe526bc058d6b9f3309215.tar.gz"
+  version "0.1.0"
+  sha256 "618cdd42d4662dcfc6ff2da25854cc0039e664df1a5caff7026acb4074c8e269"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+
+  def install
+    mkdir "c-build" do
+      system "cmake", "..", "-DROBOTS_BUILD_TESTS=ON", *std_cmake_args
+      system "make"
+
+      bin.install "robots"
+
+      if OS.mac?
+        lib.install "librobots.dylib"
+        require "macho"
+        MachO::Tools.add_rpath(bin/"robots", lib.to_s)
+      elsif OS.linux?
+        mv "librobots.so", "/usr/lib/librobots.so"
+      end
+    end
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "c-build", "-DCMAKE_INSTALL_RPATH=#{rpath}", *std_cmake_args
    system "cmake", "--build", "c-build"
    system "cmake", "--install", "c-build"
```

1. Follow the pattern used in other formulae.
2. Don't build the tests.
3. Use `CMAKE_INSTALL_RPATH` so you don't need to adjust the RPATH manually.

Reply:
Fixed. Thank you!